tests/client_test.py

```diff
@@ -19,6 +19,11 @@ def test_client_get(self):
         self.assertEqual(str(ds), expect)
         self.assertEqual(ds.structure.format, 'csv')
 
+    def test_client_get_readme(self):
```

Comment:
Happy to move this test into the above one if you don't think it merits its own?

Reply:
It's good right here, I like to put more specific tests (get readme) after the more general ones (get).